### DISCUSSION TITLE
Is it possible to submit modules to mini.nvim

### DISCUSSION THREAD

**Discussion by saccarosium** [nvim-mini/mini.nvim#495] [General]
Hi, I was wondering if it is possible to submit a module to mini.nvim. I saw in the planned module in the README that you are planing for an alternative to `ThePrimegean/harpoon`. I've created a minimal version called [neomarks](https://github.com/saccarosium/neomarks) and I will love to have it in mini.nvim. 

Obviously I will maintained myself and change the API and documentation to match other module.

**Comment by echasnovski** [comment]
Hi!

This is not impossible. However, the overall development of 'mini.nvim' has reached the point when introducing new module is tied with a lot of tangential work (like designing whole API to be consistent with other modules, writing tests with 'mini.test', documentation with 'mini.doc', proper README aligned with other modules, etc.). Thus I'd rather not outsource the whole task of creating module from scratch.

I am really grateful for the initiative though and will definitely look into 'neomarks' when I will write 'mini.places' (with proper addition to "similar plugins" section, of course).
